planetarypraxis/smartforests
# Tag fixes

Downstream from PR #110 

- Fixes the display of Logbook page tags, to correctly include child LogbookEntry tags (closing #106). 
- Standardises an `all_tags` property across the different page classes for use in the article metadata template, which seems to have fixed #66 as well.
- Adds missing language code variable required for linking LogbookEntry tags to the sidepanel, fixing #112.
- Moves filtering UI to a sidepanel, which fixes #86 #87
- Closes #117 

## How Can It Be Tested?

- Open each of the page types and you should see that tags are showing up.
- On Logbook pages, tags of child Entries should also show up.
- No tags should show up more than once.
- On index pages, the filter UI should appear as a sidepanel and work correctly

## Types of changes


## Checklist:
